controllers/proverbs-controllers.js

```diff
@@ -0,0 +1,73 @@
+const { validationResult } = require('express-validator');
+
+const Proverb = require('../models/proverb');
+const user = require('../models/user');
+
+const getProverbs = async (req, res, next) => {
+	res.send('hi from prov bro ;) ');
+};
+
+const postProverb = async (req, res, next) => {
+	const errors = validationResult(req);
+	if (!errors.isEmpty()) {
+		return next(errors);
+	}
+
+	const { proverb, literalTranslation: translation, explanation, userId } = req.body;
+
+	const postedProverb = new Proverb({
+		proverb,
+		translation,
+		explanation,
+		userId
+	});
+
+	if (userId) {
+		let user;
+		try {
+			user = await user.findById(userId);
+		} catch (error) {
+			console.log(error);
+			res.status(500).json({
+				msg: 'Could not find user in database'
+			});
+			return next(error);
+		}
+
+		if (!user) {
+			res.status(500).json({
+				msg: 'Could not find user'
+			});
+			throw new Error('Invalid credentials.');
+		}
+		console.log(user);
```

Comment:
did you use this console.log for debugging? if yes, it is not useful anymore so you can remove it